src/accel/cuda/GPUSolver.cu

```diff
@@ -693,27 +686,28 @@ FP_PRECISION GPUSolver::getFSRSource(int fsr_id, int group) {
              _num_groups * sizeof(FP_PRECISION),
              cudaMemcpyDeviceToHost);
 
-  /* Initialize variables */
   FP_PRECISION fission_source = 0.0;
   FP_PRECISION scatter_source = 0.0;
   FP_PRECISION total_source;
 
-  /* Compute total fission source for current region */
-  for (int e=0; e < _num_groups; e++) {
-    fission_source += fsr_scalar_fluxes[e] * nu_sigma_f[e];
-  }
-
-  fission_source /= _k_eff;
-
-  /* Compute total scattering source for this FSR */
+  /* Compute total scattering and fission sources for this FSR */
   for (int g=0; g < _num_groups; g++) {
     scatter_source += sigma_s[(group-1)*(_num_groups)+g] 
-                    * fsr_scalar_fluxes[g];
+                      * fsr_scalar_fluxes[g];
```

Comment:
What is the default? Emacs does have one but I find myself disagreeing with
it quite often. But that's not a good reason. If you like vim's default,
add it to the style guide and I'll start abiding by it.
On Jul 27, 2015 6:56 PM, "Geoffrey Gunow" <notifications@github.com> wrote:

> In src/accel/cuda/GPUSolver.cu
> <https://github.com/mit-crpg/OpenMOC/pull/186#discussion_r35608087>:
>
> >    for (int g=0; g < _num_groups; g++) {
> >      scatter_source += sigma_s[(group-1)*(_num_groups)+g]
> > -                    * fsr_scalar_fluxes[g];
> > +                      * fsr_scalar_fluxes[g];
>
> I really don't have a preference too much, except Vim I believe does have
> a default (I'm not sure about Emacs). I would opt for whatever the default
> is.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/186/files#r35608087>.
>